### PR TITLE
reduce bigquery frequency

### PR DESCRIPTION
**Pull request by BenTheElder**
ref: https://kubernetes.slack.com/archives/C09QZ4DQB/p1677608599806579

### CODE REVIEW

File: config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-test-infra.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ postsubmits:
 
 periodics:
 - name: metrics-bigquery
-  cron: "03 0-23/3 * * *"  # like interval: 6h, but explicitly starting at 00:03 UTC
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-test-infra.yaml:32]: AFAICT this was actually every 3 hours https://elmah.io/tools/cron-parser/#03_0-23/3_*_*_*

Comment (dims) [reply config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-test-infra.yaml:32]: comment drift :)

**Comment by BenTheElder** [comment]
Screenshot 2023-03-01 at 10 04 36 AM

looking good so far, we'll have better data tomorrow though when we've had a full day

**Comment by BenTheElder** [comment]
It dropped to $8.35 for the first full day with this change. One of these jobs went from running 24 times to twice daily.

**Pull request merged**